pkg/controller/volume/scheduling/scheduler_binder_test.go

```diff
@@ -2010,3 +2130,114 @@ func TestFindAssumeVolumes(t *testing.T) {
 		testEnv.validatePodCache(t, testNode.Name, pod, expectedBindings, nil)
 	}
 }
+
+// TestCapacity covers different scenarios involving CSIStorageCapacity objects.
+// Scenarios without those are covered by TestFindPodVolumesWithProvisioning.
+func TestCapacity(t *testing.T) {
```

Comment:
Can you also add some integration test cases so we test the full interaction with etcd?

https://github.com/kubernetes/kubernetes/blob/master/test/integration/volumescheduling/volume_binding_test.go

Reply:
Okay, done.